winrt/test.internal/graphics/CanvasTextRendererUnitTests.cpp

```diff
@@ -219,7 +219,7 @@ namespace canvas
 				for (int i = 0; i < 3; ++i)
 				{
 					int inc = i * 4;
-					Assert::AreEqual(inc + 1, glyphs[i].Index);
+          Assert::AreEqual(inc + 1, glyphs[i].Index);
```

Comment:
To make this delta a little bit cleaner, would it be possible to revert this file (no whitespace change)?